pkg/resource/pod_distruption_budget_test.go

```diff
@@ -0,0 +1,71 @@
+package resource_test
+
+import (
+	"testing"
+
+	"fmt"
+
+	"github.com/cockroachdb/cockroach-operator/pkg/labels"
+	"github.com/cockroachdb/cockroach-operator/pkg/resource"
+	"github.com/cockroachdb/cockroach-operator/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	policy "k8s.io/api/policy/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func TestPDBBuilder(t *testing.T) {
+
+	var MaxInt int32 = 1
+	cluster := testutil.NewBuilder("test-cluster").Namespaced("test-ns").WithMaxUnavaible(&MaxInt)
+	commonLabels := labels.Common(cluster.Cr())
+
+	labelSelector, err := metav1.ParseToLabelSelector("app=" + cluster.Cr().Name)
+	if err != nil {
+		t.Errorf("unexpected error parsing label: %v", err)
+	}
+
+	max := intstr.FromInt(1)
```

Comment:
I would suggest to consider a better name here as well as max is also kind of unclear.

Reply:
changed